src/server/pps/cmds/cmds.go

```diff
@@ -266,16 +309,29 @@ $ {{alias}} -p foo -i bar@YYY`,
 					})
 				} else {
 					// We are listing all sub-jobs, possibly restricted to a single pipeline
+					var pipeline *pps.Pipeline
+					if pipelineName != "" {
+						pipeline = &pps.Pipeline{Name: pipelineName}
```

Comment:
Don’t forget to set the project too: `Project: &pfs.Project{Name: project}` — this is another reason to only support a single project.